File: Day1/java-with-bazel/src/test/java/org/tektutor/HelloTest.java
package org.tektutor;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class HelloTest {

	@Test
	public void sayHelloTest() {
		Hello hello = new Hello();

		String expectedResult = "Hello, Bazel !";
		String actualResult = hello.sayHello();

		assertEquals( expectedResult, actualResult );
	}

}

File: Day3/code-coverage/src/hello_test.cpp
#include <gtest/gtest.h>
#include "hello.h"

TEST(HelloTest, Greet) {
    Hello h;
    std::string msg = "World";
    EXPECT_EQ(h.sayHello(msg), "Hello, World!");

    msg = "Bazel";
    EXPECT_EQ(h.sayHello(msg), "Hello, Bazel!");
}
